### PR TITLE
Add link to Location Details page in map popup

### LINKED ISSUE #168: Update map popups to include link to location details page
This is where they can add assets

### PR DESCRIPTION
**Pull request by mateoclarke**
Screen Shot 2019-08-01 at 7 27 35 PM

Closes #168

### CODE REVIEW

**Comment by mateoclarke** [comment]
@dianamartin this has now been updated in PROD.

Aug-01-2019 19-58-19

**Comment by dianamartin** [comment]
@mateoclarke that’s so awesome! You’ve made the sign technicians very happy.

File: src/SelectLocation.js [hunk 1]
```diff
@@ -612,6 +616,17 @@ export default class SelectLocation extends Component {
                   offset={{ bottom: [0, -40] }}
                 >
                   <div className="container popup">
+                    <span>
+                      <a
+                        href={`https://atd.knack.com/signs-markings#work-order-signs/view-work-orders-details-sign/${workOrderId}/view-work-order-signs-location-details/${
+                          activeSign.id
+                        }`}
+                        target="_top"
```
Comment (mddilley) [inline src/SelectLocation.js]: 🎯

Comment (mateoclarke) [reply src/SelectLocation.js]: @sergiogcx you were right!!

**Pull request merged**